### PR TITLE
EES-4289 Update python dependencies

### PR DESCRIPTION
**Pull request by markhiveit**
This PR updates the python dependencies.

When running the UI tests you may now see these warnings:
```
/home/mark/.local/share/virtualenvs/explore-education-statistics-a7wW707F/lib/python3.10/site-packages/robot/utils/argumentparser.py:80: UserWarning: ArgumentParser option 'auto_pythonpath' is deprecated since Robot Framework 5.0.
  warnings.warn("ArgumentParser option 'auto_pythonpath' is deprecated "
/home/mark/.local/share/virtualenvs/explore-education-statistics-a7wW707F/lib/python3.10/site-packages/robot/running/builder/builders.py:112: UserWarning: 'TestSuiteBuilder' argument 'included_suites' is deprecated and has no effect. Use the new 'included_files' argument or filter the created suite instead.
  warnings.warn("'TestSuiteBuilder' argument 'included_suites' is deprecated "
```

This is a known issue with pabot that has yet to be fixed: https://github.com/mkorpela/pabot/issues/539

### Other changes:
- In `create_snapshots.py`, I've made it so the user must provide either one of the `--validate` or `--ci` options. Previously, if you didn't provide an option the script wouldn't do much.
- Updated a couple of scripts/READMEs to use the new `FeaturedTables` table instead the old and removed `DataBlock_HighlightName` column in the `ContentBlock` table.
- Increased `csv.field_size_limit` as when running the visual testing scripts they were failing to run.

### CODE REVIEW

File: tests/robot-tests/scripts/get_auth_tokens.py [hunk 1]
```diff
@@ -144,7 +142,7 @@ def get_identity_info(
     chrome_options.add_argument("--ignore-certificate-errors")
     chrome_options.add_argument("--disable-logging")
     chrome_options.add_argument("--log-level=\3")
-    driver = webdriver.Chrome(options=chrome_options, desired_capabilities=DesiredCapabilities.CHROME)
```
Comment (luke-h1) [inline tests/robot-tests/scripts/get_auth_tokens.py:147]: Why has this been removed?

File: tests/robot-tests/scripts/run_tests_pipeline.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def run_tests_pipeline():
         raise Exception(f"Invalid environment provided: {args.env}. Valid environments: {valid_environments}")
 
     subprocess.check_call(["google-chrome-stable", "--version"])
+    subprocess.check_call(["python", "--version"])
```
Comment (luke-h1) [inline tests/robot-tests/scripts/run_tests_pipeline.py:26]: No need for this check afaik. We install a hardcoded python version via a pipeline task so not sure why this is needed

File: tests/robot-tests/scripts/get_auth_tokens.py (hunk 1 above)
Comment (markhiveit) [reply tests/robot-tests/scripts/get_auth_tokens.py:147]: Desired capabilities is no longer a thing in Selenium. Things it previously configured can be changed via the options object:
https://support.smartbear.com/crossbrowsertesting/docs/automated-testing/frameworks/selenium/about/use-of-desiredcapabilities-has-been-deprecated.html

File: tests/robot-tests/scripts/run_tests_pipeline.py (hunk 2 above)
Comment (markhiveit) [reply tests/robot-tests/scripts/run_tests_pipeline.py:26]: I added it as more of a convenience, to save having to click around. I'm going to leave it, but if others feel strongly about it, I'll remove it.

**Pull request merged**